src/install_host_app.sh

```diff
@@ -126,16 +140,26 @@ else
   curl -sSL "$MANIFEST_URL" > "$MANIFEST_FILE_PATH"
 fi
 
-# When using sed on macOS, backup extension is an mandatory argument
-#   whereas on GNU sed or BSD sed backup extension may be omit.
-if [ "$KERNEL_NAME" == 'Darwin' ]; then
+if [ "$IS_BSD" = true ]; then
+  # Use BSD style sed on BSD-ish systems
   # Replace path to python3 executable
   /usr/bin/sed -i '' "1 s@.*@#\!${PYTHON3_PATH}@" "$HOST_FILE_PATH"
+  # Replace path to pass (only in a line starting with `COMMAND =`)
+  /usr/bin/sed -i '' "/^COMMAND *=/s@\"pass\"@\"$PASS_PATH\"@" "$HOST_FILE_PATH"
+  # Set the PATH to match this script's
+  /usr/bin/sed -i '' "s@\"PATH\":.*@\"PATH\": \"$PATH\"@" "$HOST_FILE_PATH"
+
```

Comment:
This is fine, but why don't we do this to set the `PATH` to the value it has when the install script is executed? I don't understand why we would replace it by an arbitrary `PATH` that we chose ourselves?!